CODEOWNERS

```diff
@@ -0,0 +1 @@
+*    @giantswarm/team-honeybadger
```

Comment:
Okay with me but do we want to get reviewers for every PR in this repo?

Reply:
I see the problem now - let's got with codeowners for now then. Pinging the team is a bit weird because I at least treat pings differently from review requests in my mails, so it would be confusing.